api/filters/nameref/nameref.go

```diff
@@ -337,19 +337,16 @@ func (f Filter) selectReferral(
 		return candidates[0], nil
 	}
 	ids := getIds(candidates)
-	f.failureDetails(candidates)
-	return nil, fmt.Errorf(" found multiple possible referrals: %s", ids)
+	return nil, fmt.Errorf(" found multiple possible referrals: %s\n%s", ids, f.failureDetails(candidates))
```

Comment:
Why is there an extra space at the beginning of this error?

Reply:
I dunno... there was before too. Probably just a typo.